components/tidb_query_expr/src/impl_string.rs

```diff
@@ -1308,76 +1308,133 @@ mod tests {
             (
                 Some(b"bar".to_vec()),
                 Some(b"foobarbar".to_vec()),
+                Collation::Utf8Mb4Bin,
                 Some(4i64),
             ),
-            (Some(b"xbar".to_vec()), Some(b"foobar".to_vec()), Some(0i64)),
-            (Some(b"".to_vec()), Some(b"foobar".to_vec()), Some(1i64)),
-            (Some(b"foobar".to_vec()), Some(b"".to_vec()), Some(0i64)),
-            (Some(b"".to_vec()), Some(b"".to_vec()), Some(1i64)),
+            (
+                Some(b"xbar".to_vec()),
+                Some(b"foobar".to_vec()),
+                Collation::Utf8Mb4Bin,
+                Some(0i64),
+            ),
+            (
+                Some(b"".to_vec()),
+                Some(b"foobar".to_vec()),
+                Collation::Utf8Mb4Bin,
+                Some(1i64),
+            ),
+            (
+                Some(b"foobar".to_vec()),
+                Some(b"".to_vec()),
+                Collation::Utf8Mb4Bin,
+                Some(0i64),
+            ),
+            (
+                Some(b"".to_vec()),
+                Some(b"".to_vec()),
+                Collation::Utf8Mb4Bin,
+                Some(1i64),
+            ),
             (
                 Some("好世".as_bytes().to_vec()),
                 Some("你好世界".as_bytes().to_vec()),
+                Collation::Utf8Mb4Bin,
                 Some(2i64),
             ),
             (
                 Some("界面".as_bytes().to_vec()),
                 Some("你好世界".as_bytes().to_vec()),
+                Collation::Utf8Mb4Bin,
                 Some(0i64),
             ),
             (
                 Some(b"b".to_vec()),
                 Some("中a英b文".as_bytes().to_vec()),
+                Collation::Utf8Mb4Bin,
                 Some(4i64),
             ),
+            // Case-sensitive cases
```

Comment:
Please remove this line since the case started from L1363 is not case sensitive.

Reply:
done

Thanks a lot~